worker/mocks_test.go

```diff
@@ -42,8 +44,17 @@ type (
 	mockMemoryManager struct{}
 
 	mockObjectStore struct {
-		mu      sync.Mutex
-		objects map[string]map[string]object.Object
+		mu           sync.Mutex
+		objects      map[string]map[string]object.Object
+		partials     map[string]mockPackedSlab
+		bufferIDCntr uint // allows marking packed slabs as uploaded
```

Comment:
I think this is only set but never really used?

Reply:
It is used actually. I use it as the bufferID which I need to make `MarkPackedSlabsUploaded` work.